x/eibc/keeper/keeper.go

```diff
@@ -108,6 +108,7 @@ func (k *Keeper) UpdateDemandOrderWithStatus(ctx sdk.Context, demandOrder *types
 // FulfillOrder should be called only at most once per order.
 func (k Keeper) FulfillOrder(ctx sdk.Context, order *types.DemandOrder, fulfillerAddress sdk.AccAddress) error {
 	order.IsFulfilled = true
+	order.FulfillerAddress = fulfillerAddress.String()
```

Comment:
seems to me we could deprecate isFulfilled and replace it with a method that does FulfillerAdddress!=""?